examples/configurable/configurable.bal

```diff
@@ -0,0 +1,26 @@
+import ballerina/io;
+
+// Initialize the required `configurable` variables with the `?`.
+// A value must be supplied for these variables in a `configuration.toml` file.
```

Comment:
The spec doesn't use the terms "required `configurable` variables" and "optional `configurable` variables" right?

Shall we rephrase these to avoid introducing new terms?

```suggestion
// `configurable` variables can be initialized with `?`.
// A value must be supplied for such variables in a `configuration.toml` file.
```